Makefile

```diff
@@ -0,0 +1,6 @@
+NAME            := lustre-ldiskfs-zfs
+PACKAGE_VERSION := 1
+DIST_VERSION    := $(PACKAGE_VERSION)
+PACKAGE_RELEASE := 4
+
+include ./include/rpm.mk
```

Comment:
So ultimately, this Makefile should probably build the tarball you need for building an RPM.